nuxeo-easyshare-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-easyshare/nuxeo-easyshare-share.html

```diff
@@ -0,0 +1,23 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="nuxeo-easyshare-share-link.html">
+<nuxeo-slot-content name="shareDialogSlot" slot="SHARE_DIALOG" order="20">
+  <template>
```

Comment:
Missing a `nuxeo-filter`here only contributing this if the document is an easyshare folder